libs/sdk/src/utils/wasm_exec.js

```diff
@@ -14,7 +14,7 @@
 	if (!globalThis.fs) {
 		let outputBuf = "";
 		globalThis.fs = {
-			constants: { O_WRONLY: -1, O_RDWR: -1, O_CREAT: -1, O_TRUNC: -1, O_APPEND: -1, O_EXCL: -1 }, // unused
+			constants: { O_WRONLY: -1, O_RDWR: -1, O_CREAT: -1, O_TRUNC: -1, O_APPEND: -1, O_EXCL: -1, O_DIRECTORY: -1 }, // unused
```

Comment:
Are the changes to this file just the latest version of wasm_exec.js from Go?

Reply:
Ummm, I think so.  I didn't change it myself.  Should I undo those?

Reply:
oh nvm, no, I forgot the nx build step copies this in.